provider/aws.go

```diff
@@ -267,12 +267,15 @@ func changesByZone(zones map[string]*route53.HostedZone, changeSet []*route53.Ch
 	for _, c := range changeSet {
 		hostname := ensureTrailingDot(aws.StringValue(c.ResourceRecordSet.Name))
 
-		zoneID, _ := zoneNameIDMapper.FindZone(hostname)
```

Comment:
`zoneNameIDMapper` seems unsed to me now. Can we remove it?

Reply:
Good call. Removed.